azurerm/data_source_batch_account_test.go

```diff
@@ -0,0 +1,50 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccDataSourceAzureRMBatchAccount_basic(t *testing.T) {
```

Comment:
it'd be good to add a complete example (with the storage account) test in here too

Reply:
agreed

Reply:
Done.